docs/app/Components/ComponentDoc/ComponentExample.js

```diff
@@ -52,10 +52,10 @@ export default class ComponentExample extends Component {
         <Grid.Column>
           <Grid>
             <Grid.Column width={12}>
-              <Header.H3 style={{ marginBottom: 0 }}>
+              {this.props.title && <Header.H3 style={{ marginBottom: 0 }}>
                 {this.props.title}
-              </Header.H3>
-              <p>{this.props.description}</p>
+              </Header.H3>}
+              {this.props.description && <p>{this.props.description}</p>}
```

Comment:
Some examples do not have a `title` or `description` as they are under the title/desc of the example immediately above it.  We needed to hide the title to prevent empty anchor links from showing up.  Also, the description created extra unnecessary whitespace.